### PR TITLE
Enable support for collecting Grace cpu metrics

### PR DESCRIPTION
**Pull request by glowkey**

### CODE REVIEW

**Review by nvvfedorov — CHANGES_REQUESTED** [review]
Please replace magic numbers with constants.

File: pkg/dcgmexporter/parser.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func extractCounters(records [][]string, c *Config) ([]Counter, error) {
 }
 
 func fieldIsSupported(fieldID uint, c *Config) bool {
-	if fieldID < 1000 {
+	if fieldID < 1000 || fieldID > 1099 {
```
[the remark this answers is not in the document]
Comment (glowkey) [reply pkg/dcgmexporter/parser.go]: Fixed

File: pkg/dcgmexporter/system_info.go [hunk 2]
```diff
@@ -146,6 +155,15 @@ func SwitchIdExists(sysInfo *SystemInfo, switchId int) bool {
 	return false
 }
 
+func CpuIdExists(sysInfo *SystemInfo, cpuId int) bool {
```
[the remark this answers is not in the document]
Comment (glowkey) [reply pkg/dcgmexporter/system_info.go]: Good call. I've renamed to "CPU".

File: pkg/dcgmexporter/system_info.go [hunk 3]
```diff
@@ -146,6 +155,15 @@ func SwitchIdExists(sysInfo *SystemInfo, switchId int) bool {
...
+func CPUIdExists(sysInfo *SystemInfo, cpuId int) bool {
+	for _, cpu := range sysInfo.CPUs {
+		if cpu.EntityId == uint(cpuId) {
+			return true
+		}
+	}
+	return false
+}
+
 func GpuInstanceIdExists(sysInfo *SystemInfo, gpuInstanceId int) bool {
```
Comment (nvvfedorov) [inline pkg/dcgmexporter/system_info.go:167]: Because we renamed `Cpu` to `CPU`, please rename `Gpu` to `GPU`.

**Review by nvvfedorov — CHANGES_REQUESTED** [review]
Rename Gpu to GPU

File: pkg/dcgmexporter/system_info.go (hunk 3 above)
Comment (glowkey) [reply pkg/dcgmexporter/system_info.go:167]: I would prefer to do that work in a separate PR since that code was not originally touched in this patch.

**Review by nvvfedorov — APPROVED** [review]
LGTM

**Pull request merged**